regexes/client/feed_readers.yml

```
@@ -113,7 +113,7 @@
   url: 'http://readkitapp.com/'
   type: 'Feed Reader App'
 
-- regex: 'Reeder(?:[/ ](\d+[\.\d]+))?'
+- regex: 'Reeder(?:[/ ](\d+[\.\d]+))'
```

could now be simplified to 
```suggestion
- regex: 'Reeder[/ ](\d+[\.\d]+)'
```